src/components/api/index.js

```diff
@@ -11,6 +11,23 @@ export const getComments = (component, story, version) => {
 
 };
 
-export const  postComment = (component, story, version, comment) => {
-	console.log(component, version, comment);
+export const postComment = ({
+  userName,
+  userEmail,
+  userComment,
+  component,
+  story,
+  version,
+}) => {
+  const url = `${config.endpoint}/${cleanComponentId(component)}`;
+  return makeRequest(url, {
+    method: 'POST',
+    body: JSON.stringify({
+      userName: userName,
+      userEmail: userEmail,
+      'comment': userComment,
+      'stateId': story,
+      'version': version || "null",
```

Comment:
can we change 'null' to 'initial'? The api call might look a bit funny with null http://locahost:3001/button/primary/null